### PR TITLE
test: refactor pmemfile-core tests

### PR DESCRIPTION
**Pull request by GBuella**
This change is [Reviewable](https://reviewable.io/reviews/marcinslusarz/nvml/57)

### CODE REVIEW

File: src/test/file_core_crash/TEST0 [hunk 1]
```diff
@@ -43,23 +43,16 @@ require_fs_type any
...
-run e crash2
-run f openclose
-
-check
+run prep
+run openclose1
+run crash1
+run openclose2
+run crash2
+run openclose3
```
Comment (marcinslusarz) [inline src/test/file_core_crash/TEST0]: You can get rid of "run" function completely.

File: src/test/unittest/Makefile [hunk 2]
```diff
@@ -41,6 +41,7 @@ TARGET = libut.a
 OBJS = ut.o ut_alloc.o ut_file.o ut_pthread.o ut_signal.o ut_backtrace.o \
 	ut_pmemfile.o
 CFLAGS = -I$(TOP)/src/include
+CFLAGS += -I$(TOP)/src/common
```
Comment (marcinslusarz) [inline src/test/unittest/Makefile]: .

File: src/test/unittest/ut_pmemfile.c [hunk 3]
```diff
@@ -36,16 +36,19 @@
 
 #include "pmemfile_test.h"
 #include "unittest.h"
+#include "util.h"
```
Comment (marcinslusarz) [inline src/test/unittest/ut_pmemfile.c]: This doesn't seem to be needed.

**Comment by GBuella** [comment]
Review status: 0 of 22 files reviewed at latest revision, 3 unresolved discussions.

---

*[src/test/file_core_crash/TEST0, line 56 at r1](https://reviewable.io:443/reviews/marcinslusarz/nvml/57#-Kbj3l-sN_OjtWmIMZPU-r1-56:-KbjnXXEK5_aHA_MDRc_:b-896fix) ([raw file](https://github.com/marcinslusarz/nvml/blob/0fd045ac33af6eab5257c53404899da1f0c7c0f5/src/test/file_core_crash/TEST0#L56)):*
<details><summary><i>Previously, marcinslusarz (Marcin Ślusarz) wrote…</i></summary><blockquote>

You can get rid of "run" function completely.
</blockquote></details>

Done.

---

*[src/test/unittest/ut_pmemfile.c, line 39 at r1](https://reviewable.io:443/reviews/marcinslusarz/nvml/57#-Kbj3l-sN_OjtWmIMZPm-r1-39:-KbjnZ4oed44C6Uoocf3:b-896fix) ([raw file](https://github.com/marcinslusarz/nvml/blob/0fd045ac33af6eab5257c53404899da1f0c7c0f5/src/test/unittest/ut_pmemfile.c#L39)):*
<details><summary><i>Previously, marcinslusarz (Marcin Ślusarz) wrote…</i></summary><blockquote>

This doesn't seem to be needed.
</blockquote></details>

Done.

---

*[src/test/unittest/Makefile, line 44 at r1](https://reviewable.io:443/reviews/marcinslusarz/nvml/57#-Kbj3l-sN_OjtWmIMZPk-r1-44:-Kbjn_E1gQnjMHt6RCfh:b-896fix) ([raw file](https://github.com/marcinslusarz/nvml/blob/0fd045ac33af6eab5257c53404899da1f0c7c0f5/src/test/unittest/Makefile#L44)):*
<details><summary><i>Previously, marcinslusarz (Marcin Ślusarz) wrote…</i></summary><blockquote>

.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/marcinslusarz/nvml/57)*

**Pull request merged**